spatial-history/index.md

```diff
@@ -30,10 +19,42 @@ It makes sense to put an introductory paragraph here, above the map and buttons.
   <div id="map_legend"></div>
 </div>
 
-# Next Section
 
-The rest of your essay should go under the map, probably.
 
-## in sections like this
-also _italics_ and __bold__ which can also be written *like this* or **like this**
-etc. Be sure to [make use of links](http://digital.hackinghistory.ca) -- that's one of the reasons we write on the Web, and I've asked you to use links for all yor footnotes/references as well.  
+
+
```

Comment:
in markdown, a whole bunch of line breaks like this will just end up rendering as *one paragraph break*. So, instead, add a `<div class="break"> & give it some css attributes in `/css/extra-styles.css`. `clear:both` should produce the effect you want.